src/features/Dashboard/components/MembershipPanel.tsx

```diff
@@ -107,23 +96,23 @@ function MembershipPanel({
                 <Typography as="div">{translateRaw('EXPIRES_ON')}</Typography>
                 <Typography as="div">{new Date(expiryDate).toLocaleDateString()}</Typography>
               </ExpiryWrapper>
-              <SLink onClick={() => history.push(ROUTE_PATHS.MYC_MEMBERSHIP.path)}>
+              <LinkApp href={ROUTE_PATHS.MYC_MEMBERSHIP.path} mt={SPACING.SM}>
                 {translateRaw('MANAGE_MEMBERSHIP')}
-              </SLink>
-              <Link
+              </LinkApp>
+              <LinkApp
                 href={getKBHelpArticle(KB_HELP_ARTICLE.MEMBERSHIP_REQUEST_REWARDS)}
-                rel="noreferrer"
+                isExternal={true}
               >
                 <SButton colorScheme={'inverted'}>{translateRaw('REQUEST_REWARDS')}</SButton>
-              </Link>
+              </LinkApp>
             </>
           )}
           {membershipState === MembershipState.NOTMEMBER && (
             <>
               <Typography as="div">{translateRaw('MEMBERSHIP_NOTMEMBER')}</Typography>
-              <SButton onClick={() => history.push(ROUTE_PATHS.MYC_MEMBERSHIP.path)}>
-                {translateRaw('BECOME_MEMBER')}
-              </SButton>
+              <LinkApp href={ROUTE_PATHS.MYC_MEMBERSHIP.path}>
+                <SButton>{translateRaw('BECOME_MEMBER')}</SButton>
```

Comment:
Styling is off on this button now. It is not wide enough.

Reply:
Done